khmer/kfile.py

```diff
@@ -111,8 +112,11 @@ def check_space(in_files, force, _testhook_free_space=None):
             sys.exit(1)
 
 
-def check_space_for_hashtable(hash_size, force, _testhook_free_space=None):
+def check_space_for_hashtable(args, hashtype, force,
+                              _testhook_free_space=None):
     """Check we have enough size to write a hash table."""
+    hash_size = khmer_args._calculate_tablesize(args, hashtype)
```

Comment:
We should unprefix `_calculate_tablesize` if it is going to be accessed outside of `khmer_args`